src/mgmt/srv_drpc.c

```diff
@@ -205,10 +205,11 @@ ds_mgmt_drpc_start_mgmt_svc(Drpc__Call *drpc_req, Drpc__Response *drpc_resp)
 			       NULL /* addr */);
 	if (rc == -DER_ALREADY)
```

Comment:
style nit - if one clause of the if statement has curly brackets, both should.

Reply:
yes was considering changing back